analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/InsecureDeserialization.cs

```diff
@@ -38,15 +39,11 @@ public sealed class InsecureDeserialization : HotspotDiagnosticAnalyzer
         private const string DiagnosticId = "S5766";
         private const string MessageFormat = "Make sure not performing data validation after deserialization is safe here.";
 
-        private static readonly DiagnosticDescriptor Rule =
-            DiagnosticDescriptorBuilder
-                .GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager)
-                .WithNotConfigurable();
+        private static readonly DiagnosticDescriptor Rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager).WithNotConfigurable();
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
-        public InsecureDeserialization()
-            : base(AnalyzerConfiguration.Hotspot)
+        public InsecureDeserialization() : base(AnalyzerConfiguration.Hotspot)
```

Comment:
This could also be like this
```suggestion
        public InsecureDeserialization() : base(AnalyzerConfiguration.Hotspot) { }
```

Reply:
I also chain via `this` now, so only one base call is left.

Reply:
The two `base` calls are fine here, the other one is empty anyway. The `this` only adds one more stackframe in this case.